### PR TITLE
Feature/bma/test pin unlock presenter

### PR DESCRIPTION
**Pull request by bmarty**
Add some preview and test on the presenter

### CODE REVIEW

File: features/lockscreen/impl/src/main/kotlin/io/element/android/features/lockscreen/impl/unlock/PinUnlockView.kt [hunk 1]
```diff
@@ -379,3 +380,24 @@ internal fun PinUnlockDefaultViewPreview(@PreviewParameter(PinUnlockStateProvide
...
+internal fun PinUnlockInAppViewLandscapePreview() {
+    ElementPreview {
+        PinUnlockView(
+            state = aPinUnlockState(),
+            isInAppUnlock = true,
+        )
+    }
+}
+
+@Composable
```
Comment (bmarty) [inline features/lockscreen/impl/src/main/kotlin/io/element/android/features/lockscreen/impl/unlock/PinUnlockView.kt:394]: Not recorded by Paparazzi :/

**Review by ganfra — APPROVED** [review]
LGTM, except looks like there is a test not passing?

**Comment by bmarty** [comment]
Weird, all is green locally. Second attempt on the CI.

**Comment by bmarty** [comment]
CI is not happy, let's resume this later.

**Pull request closed without merging**